lib/node_modules/@stdlib/stats/base/ndarray/srange/README.md

```diff
@@ -0,0 +1,115 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# srange
+
+> Compute the [range][range] of a one-dimensional single-precision floating-point ndarray.
+
+<section class="intro">
+
+The [**range**][range] is defined as the difference between the maximum and minimum values.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var srange = require( '@stdlib/stats/base/ndarray/srange' );
+```
+
+#### srange( arrays )
+
+Computes the [range][range] of a one-dimensional single-precision floating-point ndarray.
+
+```javascript
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+
+var xbuf = [ 1.0, 3.0, 4.0, 2.0 ];
```

Comment:
Same comments as `drange`. You need to be providing `Float32Array` instances for the underlying buffer. This should be fixed throughout this PR.